src/applications/vaos/actions/newAppointment.js

```diff
@@ -1,5 +1,11 @@
+import newAppointmentFlow from '../newAppointmentFlow';
+
 export const FORM_DATA_UPDATED = 'newAppointment/FORM_DATA_UPDATED';
 export const FORM_PAGE_OPENED = 'newAppointment/FORM_PAGE_OPENED';
+export const FORM_PAGE_NAVIGATE_STARTED =
```

Comment:
These actions are so that we can show a loading spinner on the form buttons while moving between pages, when we add async data fetching for things like the CC eligibility or direct scheduling eligibility.